instagram/src/components/PostsContainer/PostContainer.js

```diff
@@ -3,28 +3,72 @@ import CommentSection from '../CommentSection/CommentSection';
 import PropTypes from 'prop-types';
 import dummyData from '../../dummy-data';
 import SearchBar from '../SearchBar/SearchBar';
+import styled from 'styled-components';
+
+//Styled Components
+const UserHeader = styled.div`
+    border: 1px solid #00618c;;
+    margin-bottom: 3%;
+    background-color: floralwhite;
+`;
+const ProfileHeader = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    padding: 3%;
+`;
+const ImgThumb = styled.img`
+    border-radius: 50%;
+`;
+const UserNamePost = styled.p`
+    font-weight: bold;
+	padding-left: 1%;
+    font-size: 1rem;
+    position: relative;
+    top: 6.5px;
+    &:hover{
+        color: #00618c;
+        cursor: pointer;
+    }
+`;
+const PostText = styled.div`
+    padding: 2%;
+`;
+const Likes = styled.p`
+    font-weight: bold;
+    padding: 2% 0;
+    
+`;
+const Stamp = styled.p`
+    color: grey;
+	font-size: .7rem;
+	margin-left: 40%;
+`;
 
  class PostContainer extends React.Component {
     constructor() {
         super();
         this.state = {
           dummyData: [],
           posts: [],
-          filteredPosts: []
+          filteredPosts: [],          
         };
       }
       componentDidMount() {
+            console.log('cdm', this.state.posts)
         this.setState({ posts: dummyData })
       }
       searchHandler = (e) => {
-          const filtered = this.state.posts.filter((post) => {
-            console.log(post)  
+          const filtered = this.state.posts.filter((post) => {  
+              console.log(post)
             return post.username.toLowerCase().includes(e.target.value.toLowerCase()) || post.likes.toString().includes(e.target.value.toString()) 
```

Comment:
Good job getting the search functionality to work and making it case insensitive